File: src/frontend/src/app/components/cadastro/setor/model/setor-grid-dto.ts
export class SetorGridDTO {
  id: string;
  nome: string;
  descricao?: string;
  centroCustoNome: string;
  deleted?: boolean;

  constructor(
    id: string,
    nome: string,
    centroCustoNome: string,
    descricao?: string,
    deleted?: boolean
  ) {
    this.id = id;
    this.nome = nome;
    this.descricao = descricao;
    this.centroCustoNome = centroCustoNome;
    this.deleted = deleted;
  }
}

File: src/backend/src/main/java/br/com/grupopipa/gestaointegrada/cadastro/setor/SetorGridDTO.java
package br.com.grupopipa.gestaointegrada.cadastro.setor;

import java.util.UUID;

import br.com.grupopipa.gestaointegrada.core.dto.GridDTO;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class SetorGridDTO implements GridDTO {
  private UUID id;
  private String nome;
  private String descricao;
  private String centroCustoNome;
  private Boolean deleted;
}
